pkg/oci/load_balancer.go

```diff
@@ -364,7 +367,7 @@ func (cp *CloudProvider) updateLoadBalancer(ctx context.Context, lb *loadbalance
 		return errors.Wrap(err, "get subnets for nodes")
 	}
 
-	actions := sortAndCombineActions(backendSetActions, listenerActions)
+	actions := sortAndCombineActions(cp.logger, backendSetActions, listenerActions)
```

Comment:
This and the one above should be context loggers I think w/ at a very minimum the lb id / service name/namespace / something.